### ISSUE TITLE
Command failure for publish

### ISSUE THREAD

**Issue by nstejaswini** [Microsoft/BotBuilder-Samples#1178]
## Sample information
1. Sample language: dotnetcore 
2. Sample name: 13.basic-bot

## Describe the bug
 An error occurs while Publishing the local project BasicBot.csproj to basicbottesting service,
Command failed: az bot publish --resource-group basicbottesting -n basicbottesting --subscription 8d39b989-548d-4757-aea          {bot} ... {json,jsonc,table,tsv,yaml,none}] [--query JMESPATH]gured default 'ws' for arg resource_group_name

## To Reproduce
Steps to reproduce the behaviour:
1. Install the Bot CLI tools.
2. Select the Azure subscription.
3. Collect the Luis Authoring Key from Luis.ai
4. Get the appID and appSecret from the Application registration portal.
6: Run the msbot clone command. Error occurs here,
      -	Command failed: az bot publish --resource-group basicbottesting -n basicbottesting --subscription 8d39b989-548d-4757-aea          {bot} ... {json,jsonc,table,tsv,yaml,none}] [--query JMESPATH]gured default 'ws' for arg resource_group_name

## Expected behaviour
The sample should work according the readme.

**Comment by mdrichardson** [comment]
I wasn't able to replicate this, even following your instructions. I *did* have an issue with the `msbot clone` command in that it required me to add `--luisAuthoringKey`. Did you have this as well? It's difficult to tell from the error you listed there.

I know some of these can't be run from a Bash prompt and must be run from PowerShell or a Command Prompt. What are you running yours in?

My successful `msbot clone` command:

![image](https://user-images.githubusercontent.com/40401643/51945127-9ca9f880-23d2-11e9-860f-1e1231ac3e7f.png)

**Comment by nstejaswini** [comment]
@sgellock The version on msbot running is  4.3.2 . I used command Prompt for running the commands.

**Comment by nstejaswini** [comment]
@mdrichardson I tried implementing it again. I did the following steps.
1.Got the path to the project folder. 
(Which is not mentioned in the readme file)
2. Install the Bot CLI tools.
3. Use az login command and choose the subscription.
 ( When I used the command 'Connect-AzureRmAccount', an error was diplayedsaying "Connect-AzureRmAccount' is not recognized as an internal or external command, operable program or batch file.")
4. Run the msbot clone command.
5. Update the "YOUR_LUIS_SERVICE_NAME" in 'BasicBot.cs'
(Didn't understand why there is Step 6.Update in `Startup.cs' in readme.)
6. Update the 'botFileSecret' in appsettings.json with the secret key obtained after running msbot clone command successfully.
7. Run the project in Visual Studio.
8. Tested in Emulator.
